tests/jwt_dump.c

```diff
@@ -148,6 +148,46 @@ START_TEST(test_jwt_dump_str)
 }
 END_TEST
 
+START_TEST(test_jwt_dump_grants_str)
+{
+	jwt_t *jwt = NULL;
+	int ret = 0;
+	char *out;
+	const char *val = NULL;
+
+	ret = test_set_alloc();
+	ck_assert_int_eq(ret, 0);
+
+	ret = jwt_new(&jwt);
+	ck_assert_int_eq(ret, 0);
+	ck_assert(jwt != NULL);
+
+	ret = jwt_add_grant(jwt, "iss", "files.maclara-llc.com");
+	ck_assert_int_eq(ret, 0);
+
+	ret = jwt_add_grant(jwt, "sub", "user0");
+	ck_assert_int_eq(ret, 0);
+
+	ret = jwt_add_grant(jwt, "ref", "XXXX-YYYY-ZZZZ-AAAA-CCCC");
+	ck_assert_int_eq(ret, 0);
+
+	ret = jwt_add_grant_int(jwt, "iat", (long)time(NULL));
+	ck_assert_int_eq(ret, 0);
+
+	out = jwt_dump_grants_str(jwt, 1);
+	ck_assert(out != NULL);
```

Comment:
Let's add an expected static string and do a `strcmp()` on it and the returned `out` for a little extra sanity check.